### PR TITLE
Feature/DTGB-452: Vanalles

### PR DESCRIPTION
**Pull request by meirege**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated the style guide CHANGELOG accordingly.
- [ ] I have updated the gent_base CHANGELOG accordingly.

### CODE REVIEW

File: styleguide/components/41-organisms/contact-box/contact-box.twig [hunk 1]
```diff
@@ -29,39 +29,69 @@
...
+      <h3>
+        <button class="accordion--button"
+                aria-expanded="false"
+                aria-controls="contact-box--content">
+          {{ 'There are 5 other contacts available' }}
+        </button>
+      </h3>
+
+      <div class="accordion--content"
+        id="contact-box--content">
```
Comment (Jeroen005) [inline styleguide/components/41-organisms/contact-box/contact-box.twig]: is de id noodzakelijk, kans is klein dat we deze uniek kunnen houden.

File: templates/core/block/block--sg-search-403-search-block.html.twig [hunk 2]
```diff
@@ -28,7 +28,8 @@
 
 {%
   set classes = [
-    'search-block'
+  'search-block',
```
Comment (Jeroen005) [inline templates/core/block/block--sg-search-403-search-block.html.twig]: indent

**Review by Jeroen005 — CHANGES_REQUESTED** [review]

File: templates/core/block/block--sg-search-403-search-block.html.twig (hunk 2 above)
Comment (meirege) [reply templates/core/block/block--sg-search-403-search-block.html.twig]: Fixed.

File: styleguide/components/41-organisms/contact-box/contact-box.twig (hunk 1 above)
Comment (meirege) [reply styleguide/components/41-organisms/contact-box/contact-box.twig]: Heb unique id's toegevoegd in de stijlgids. De drupal templates krijgen ook een unique id nu.

File: styleguide/components/61-layouts/general/detail-layout/_detail-layout.scss [hunk 3]
```diff
@@ -8,7 +8,11 @@
   // Give each direct child a specific width.
   // This is a fallback width given to all children of the detail layout.
   > * {
-    margin-bottom: 2rem;
+    & {
```
Comment (Jeroen005) [inline styleguide/components/61-layouts/general/detail-layout/_detail-layout.scss]: strange? klopt dit?

**Review by Jeroen005 — CHANGES_REQUESTED** [review]

File: styleguide/components/61-layouts/general/detail-layout/_detail-layout.scss (hunk 3 above)
Comment (meirege) [reply styleguide/components/61-layouts/general/detail-layout/_detail-layout.scss]: Ja, dit target alle direct children van een .detail-layout maar niet headings.

Comment (meirege) [reply styleguide/components/61-layouts/general/detail-layout/_detail-layout.scss]: Nvm... je had gelijk...

**Review by Jeroen005 — APPROVED** [review]

**Pull request merged**